content/pt-br/docs/concepts/configuration/windows-resource-management.md

```diff
@@ -0,0 +1,75 @@
+---
+reviewers:
+- jayunit100
+- jsturtevant
+- marosset
+- perithompson
+title: Gerenciamento de recursos para nós do Windows
+content_type: concept
+weight: 75
+---
+
+<!-- overview -->
+
+Esta página descreve as diferenças em como os recursos são gerenciados entre o Linux e o Windows.
+
+<!-- body -->
+
+Em nós do Linux, {{< glossary_tooltip text="cgroups" term_id="cgroup" >}} são usados como um limite de pod para controle de recursos. 
+Os contêineres são criados dentro desse limite para o isolamento de rede, processo e sistema de arquivos. 
+As APIs Linux cgroup podem ser usadas para coletar estatísticas de uso de CPU, E/S e memória.
+
+Em contraste, o Windows usa um [_objetos de trabalho_](https://docs.microsoft.com/windows/win32/procthread/job-objects) por contêiner com um filtro de namespace do sistema
+para conter todos os processos em um contêiner e fornecer isolamento lógico ao hospedar.
+(Os objetos de trabalho são um mecanismo de isolamento de processo do Windows e são diferentes dos
+que o Kubernetes chama de {{< glossary_tooltip term_id="job" text="Job" >}}).
+
+Não há como executar um contêiner do Windows sem a filtragem de namespace. 
+Isso significa que os privilégios do sistema não podem ser declarados no contexto do host e, 
+portanto, os contêineres privilegiados não estão disponíveis no Windows.
+Os contêineres não podem assumir uma identidade do host porque o Gerente de conta de segurança (SAM) é separado.
+
+## Gerenciamento de memória {#resource-management-memory}
+
+O Windows não possui um eliminador de processo de falta de memória como o Linux. 
```

Comment:
Suggestion: "eliminador de processo por falta de memória", instead of "eliminador de processo de falta de memória".